sormas-backend/src/main/java/de/symeda/sormas/backend/person/PersonFacadeEjb.java

```diff
@@ -379,7 +379,10 @@ public void validate(PersonDto source) throws ValidationRuntimeException {
 			cq.where(filter);
 		}
 
-		cq.multiselect(personJoin.get(Person.UUID), contactRoot.get(Contact.FOLLOW_UP_UNTIL));
+		final Expression<Object> followUpStatusExpression = cb.selectCase()
```

Comment:
There also is a getCaseLatestFollowUpEndDates(...) in this class, which needs this fix, too.